docs/README-ptbr.md

```diff
@@ -0,0 +1,80 @@
+<p>
+  <a href="https://svelte.dev">
+	<img alt="Cybernetically enhanced web apps: Svelte" src="https://svelte-assets.surge.sh/banner.png">
+  </a>
+
+  <a href="https://www.npmjs.com/package/svelte">
+    <img src="https://img.shields.io/npm/v/svelte.svg" alt="npm version">
+  </a>
+
+  <a href="https://packagephobia.now.sh/result?p=svelte">
+    <img src="https://packagephobia.now.sh/badge?p=svelte" alt="install size">
+  </a>
+
+  <a href="https://travis-ci.org/sveltejs/svelte">
+    <img src="https://api.travis-ci.org/sveltejs/svelte.svg?branch=master"
+         alt="build status">
+  </a>
+
+  <a href="https://github.com/sveltejs/svelte/blob/master/LICENSE">
+    <img src="https://img.shields.io/npm/l/svelte.svg" alt="license">
+  </a>
+</p>
+
+## O que é Svelte?
+
+Svelte é um novo jeito de construir aplicações web. É um compilador que pega seus componentes declarativos e os converte em JavaScript eficiente que atualiza com grande precisão o DOM.
```

Comment:
Instead of 

> que atualiza com grande precisão o DOM

it could be

> que atualiza o DOM cirurgicamente

Reply:
Neste caso, cirurgicamente seria muito literal/ao pé da letra! Mas também funciona